_posts/2023/2023-08-24-toolbox.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Toolbox Workshop 2023
+categories:
+  - special
+  - toolbox
+  - veranstaltungen
+
+image: /images/toolbox.png
+author: Christian
+---
+
+In den beiden letzten Wochen der Sommer-Semesterferien veranstaltet PeP et al. wieder den Toolbox-Workshop.
+Die [Anmeldung](https://registration.pep-dortmund.org/events/toolbox23/registration/) ist erforderlich,
+da wir über diesen Weg alle weiteren Informationen verteilen werden.
+
+Der Workshop richtet sich vor allem an die kommenden Drittsemester
+und behandelt die Grundlagen des wissenschaftlichen Arbeitens
+und den Umgang mit den nötigen Werkzeugen.
+
+Vom 25. bis zum 29. September werden die Themen  
+„Grundlagen der Programmiersprache Python”, „Versuche auswerten und erste Datenanalyse mit NumPy und SciPy”, „Grafiken erstellen mit matplotlib”, „Die Unix-Kommandozeile”, „Automatisieren mit Gnu Make” und „Versionskontrolle mit Git” besprochen.

```

Comment:
Kann man das nicht in Textfassen statt in "Titel"?
Vom 25. bis zum 29. September werden Grundlagen der Datenanalyse in der Programmiersprache Python (NumPy, SciPy, matpltotlib) sowie der Umgang mit der Kommandozeile und einigen weiteren Tools des wissenschaftlichen Arbeitens (Gnu Make, Git) interaktiv vorgestellt.